docs/README.raspberrypi

```diff
@@ -39,24 +39,17 @@ build for the first Raspberry Pi, the commands have to be adapted to use
 
     $ cd xbmc/tools/depends
     $ ./bootstrap
-    $ PATH="$PATH:/opt/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin" \
-       ./configure --host=arm-linux-gnueabihf \
+    $ ./configure --host=arm-linux-gnueabihf \
        --prefix=/opt/kodi-bcm/kodi-dbg \
-       --with-toolchain=/usr/local/bcm-gcc/arm-bcm2708hardfp-linux-gnueabi/sysroot \
+       --with-toolchain=/opt/gcc-linaro-arm-linux-gnueabihf-raspbian-x64 \
```

Comment:
I'd suggest changing this to:
``--with-toolchain=/opt/arm-rpi-4.9.3-linux-gnueabihf \``
Assuming you are referring to tools from https://github.com/raspberrypi/tools/tree/master/arm-bcm2708
That is a newer 4.9.3 toolchain (vs the 4.8.3 prerelease one from gcc-linaro-arm-linux-gnueabihf-raspbian-x64) that is used when building the pi kernel and /opt/vc/lib libraries.